static/app/components/events/highlights/highlightsIconSummary.spec.tsx

```diff
@@ -100,11 +100,41 @@ describe('HighlightsIconSummary', function () {
     expect(screen.getAllByRole('img')).toHaveLength(4);
   });
 
+  it('hides client_os and browser contexts for Meta-Framework backend issues', function () {
+    const duplicateOsContextEvent = EventFixture({
+      contexts: {
+        client_os: {
+          type: 'os',
+          name: 'macOS',
+          version: '15.3',
+        },
+        os: {
+          type: 'os',
+          name: 'Linux',
+          version: '5.10.243',
+        },
+        runtime: {
+          name: 'node',
+          runtime: 'node v20.18.3',
+          type: 'runtime',
+          version: 'v20.18.3',
+        },
+      },
+    });
+    render(<HighlightsIconSummary event={duplicateOsContextEvent} group={group} />);
+    expect(screen.getByText('Linux')).toBeInTheDocument();
+    expect(screen.getByText('5.10.243')).toBeInTheDocument();
+    expect(screen.getByText('node')).toBeInTheDocument();
+    expect(screen.getByText('v20.18.3')).toBeInTheDocument();
+    expect(screen.queryByText('macOS')).not.toBeInTheDocument();
+    expect(screen.queryByText('15.3')).not.toBeInTheDocument();
+  });
+
   it('deduplicates client_os and os contexts', function () {
     const duplicateOsContextEvent = EventFixture({
       contexts: {
         client_os: {
-          type: 'client_os',
+          type: 'os',
```

Comment:
Maybe I'm missing something here but it seems like `client_os` is really only an alias and is never used as a type. At least Relay and the SDK only use `os`

Reply:
I'll need to double check this, what the spec says, if this is a bug in Relay or if this is actually valid according to the spec (having a key different from the type).

Reply:
Okay, so this is actually correct, nice:

> The contexts type can be used to define arbitrary contextual data on the event. It accepts an object of key/value pairs. The key is the “alias” of the context and can be freely chosen. However, as per policy, it should match the type of the context unless there are two values for a type. You can omit type if the key name is the type.

https://develop-git-python3bydefault.sentry.dev/sdk/event-payloads/contexts/

Type should be `os`, key `client_os`.